x/issuance/types/msg_test.go

```diff
@@ -70,25 +71,13 @@ func (suite *MsgTestSuite) TestMsgIssueTokens() {
 			args{
 				sender:   suite.addrs[0],
 				tokens:   sdk.NewCoin("valid", sdk.NewInt(100)),
-				receiver: sdk.AccAddress{},
+				receiver: "",
 			},
 			errArgs{
 				expectPass: false,
 				contains:   "receiver address cannot be empty",
 			},
 		},
-		{
-			"invalid tokens",
-			args{
-				sender:   suite.addrs[0],
-				tokens:   sdk.Coin{Denom: "Invalid", Amount: sdk.NewInt(100)},
-				receiver: suite.addrs[1],
-			},
-			errArgs{
-				expectPass: false,
-				contains:   "invalid tokens",
-			},
-		},
```

Comment:
Any reason for removing the test? The input coin field could still be invalid event though they relaxed the denom constraints. I'd change the test to create an invalid coin in a different way.